client/jira.go

```diff
@@ -75,8 +76,28 @@ type (
 	AssignRequest struct {
 		Name string `json:"name,omitempty"`
 	}
+
+	TransitionsResult struct {
+		Transitions []Transition `json:"transitions"`
+	}
+
+	Transition struct {
+		Name string `json:"name"`
+	}
 )
 
+func GetTransitions(issueId string) (TransitionsResult, error) {
+	var results TransitionsResult
+	path := fmt.Sprintf("/rest/api/2/issue/%s/transitions", issueId)
+	request, err := constructGetRequest(path)
+	if err != nil {
+		return results, err
+	}
+	statusCode, err := SendRequest(request, &results)
+	log.Printf("The results (tranisions) are %v and the status code is %d", results, statusCode)
```

Comment:
would remove this, we're using the standard Go logger so we don't have any debug options sadly :( could be a good issue to write to improve this project.